### DISCUSSION TITLE
Mention extra exports in the documentation

### DISCUSSION THREAD

**Discussion by bartbutenaers** [QuorumDMS/ftp-srv#332] [Ideas]
Dear,

In the [example](https://github.com/QuorumDMS/ftp-srv/tree/main#usage) on the readme page, the following statement is being used:
```
return reject(new errors.GeneralError('Invalid username or password', 401))
```
That error class is being implemented in [errors.js](https://github.com/QuorumDMS/ftp-srv/blob/main/src/errors.js#L2), so I get this exception when running this code snippet:
> ReferenceError: errors is not defined

After some digging in this repository, I found that the errors are exported [here](https://github.com/QuorumDMS/ftp-srv/blob/main/ftp-srv.js#L8):
```
module.exports.ftpErrors = errors;
```
So my problem was solved by rewriting the statement in the example code like this:
```
return reject(new FtpServer.ftpErrors.GeneralError('Invalid username or password', 401));
```
It would be nice if somebody could adjust the readme page to explain how the imports can be used. 
 
Can create a pull request myself, but then it will have to wait until my [other](https://github.com/QuorumDMS/ftp-srv/pull/331) PR is merged (because I made those changes on my fork's master branch...).

Thanks!!
Bart

**Comment by psnehanshu** [comment]
This line doesn't work in typescript.

```ts
return reject(new FtpServer.ftpErrors.GeneralError('Invalid username or password', 401));
```
